src/Components/Facility/TriageForm.tsx

```diff
@@ -242,19 +242,13 @@ export const TriageForm = (props: triageFormProps) => {
     });
   };
 
-  const handleChange = (e: any) => {
-    const form = { ...state.form };
-    form[e.target.name] = e.target.value;
-    dispatch({ type: "set_form", form });
-  };
-
   if (isLoading) {
     return <Loading />;
   }
 
   return (
     <div className="px-2">
```

Comment:
Remove the padding above <Page> component. This is so that other pages are inline with this and has consistent layout.